services/web/client/Makefile

```diff
@@ -31,11 +31,17 @@ compile-dev: qx_packages ## qx compiles host' 'source' -> host's 'source-output'
 .PHONY: compile touch
 compile: ## qx compiles host' 'source' -> image's 'build-output'
 	# qx compile 'source' within $(docker_image) image  [itisfoundation/qooxdoo-kit:${QOOXDOO_KIT_TAG}]
-	@docker build -f $(docker_file) -t $(docker_image) --build-arg tag=${QOOXDOO_KIT_TAG} --target=build-client .
+	@docker build --file $(docker_file) --tag $(docker_image) \
+		--build-arg tag=${QOOXDOO_KIT_TAG} \
+		--build-arg tag=${VCS_REF} \
+		--build-arg tag=${VCS_REF_CLIENT} \
+		--build-arg tag=${VCS_STATUS_CLIENT} \
+		--build-arg tag=${VCS_URL} \
+		--target=build-client .
 
 touch: ## minimal image build with /project/output-build inside
 	# touch /project/output-build such that multi-stage 'services/web/Dockerfile' can build development target (fixes #1097)
-	@docker build -f $(docker_file) -t $(docker_image) --build-arg tag=${QOOXDOO_KIT_TAG} --target=touch .
+	@docker build --file $(docker_file) --tag $(docker_image) --build-arg tag=${QOOXDOO_KIT_TAG} --target=touch .
```

Comment:
did you check if adding the arguments here is also necesary?

Reply:
I see this touch is used from the main Makefile when doing build-devel, but what does it do? why does it call touch and then compile-dev?